### PR TITLE
Added keyboard shortcuts, addresses issue #168

### PR DESCRIPTION
**Pull request by manderso7**
And additional changes to the keyboard shortcuts section, including a link to a youtube video showing the window switcher in use, as well as a blurb at the bottom of the keyboard shortcuts section describing where to change those shortcuts.

### CODE REVIEW

File: data/gettingstarted.html [hunk 1]
```diff
@@ -1233,6 +1237,46 @@ <h3>&zwnj;Desktop shortcuts&zwnj;</h3>
                 </td>
                 <td>&zwnj;Open Applications menu&zwnj;</td>
               </tr>
+              <tr>
+                <td>
+                  <b class="btn btn-xs disabled">&zwnj;Shift&zwnj;</b>
```
Comment (fossfreedom) [inline data/gettingstarted.html]: Shift + Shift + Printscreen ?

File: data/gettingstarted.html [hunk 2]
```diff
@@ -1281,7 +1325,11 @@ <h3>&zwnj;Window shortcuts - general&zwnj;</h3>
                   <i class="material-icons">&#xE145;</i>
                   <b class="btn btn-xs disabled">S</b>
                 </td>
-                <td>&zwnj;Call Window Shuffler gui (window tiling grid).&zwnj;</td>
+                <td>&zwnj;Call Window Shuffler gui (window tiling grid).&zwnj;
```
Comment (fossfreedom) [inline data/gettingstarted.html]: Formatting has gone a bit awry.  If you are using vscode to edit - then I would recommend the beautify extension to reformat the code.

File: data/gettingstarted.html [hunk 3]
```diff
@@ -1281,7 +1325,11 @@ <h3>&zwnj;Window shortcuts - general&zwnj;</h3>
...
+<ul>
+  <li><a onclick="cmd('link?https://www.youtube.com/watch?v=Ii4n8H4XEzc&feature=youtu.be')">&nbsp;Window Shuffler Demonstration&nbsp;</a></li>
```
Comment (fossfreedom) [inline data/gettingstarted.html]: Would be better to have a button here as per https://github.com/UbuntuBudgie/budgie-welcome/blob/master/data/store.html#L30

File: data/gettingstarted.html [hunk 4]
```diff
@@ -1559,6 +1607,13 @@ <h3>&zwnj;Window shortcuts - numeric keypad&zwnj;</h3>
                   </td>
                   <td>&zwnj;Toggle maximize.&zwnj;</td>
                 </tr>
+                             <div class="1904 hidden">
+              <tr>
+               <td>
+                 <b>Find and change these shortcuts in the Settings App, in the Devices -> Keyboards section.</b>
```
Comment (fossfreedom) [inline data/gettingstarted.html]: Nice idea - would be even better to include a button here to navigate direct i.e. run the command `gnome-control-center keyboard`

i.e. similar to https://github.com/UbuntuBudgie/budgie-welcome/blob/master/data/gettingstarted.html#L1016

The "cmd" command execute's this bit of code https://github.com/UbuntuBudgie/budgie-welcome/blob/master/budgie-welcome#L324 - so need to add a new command

    elif uri == 'keyboard':
        subprocess.Popen(['gnome-control-center', 'keyboard'])

File: data/gettingstarted.html [hunk 5]
```diff
@@ -1233,6 +1237,46 @@ <h3>&zwnj;Desktop shortcuts&zwnj;</h3>
...
+                </td>
+                 <td>&zwnj;Copy a screenshot of an area to clipboard&zwnj;</td>
+              </tr>
+              <tr>
+                <td>
+                  <b class="btn btn-xs disabled">&zwnj;Shift&zwnj;</b>
+                  <i class="material-icons">&#xE145;</i>
+                  <b class="btn btn-xs disabled">Print</b>
+                </td>
+                <td>&zwnj;Save a screenshot of an area to Pictures&zwnj;</td>
```
Comment (fossfreedom) [inline data/gettingstarted.html]: "Pictures folder" would be easier to understand I would suggest

File: data/gettingstarted.html [hunk 6]
```diff
@@ -1233,6 +1237,46 @@ <h3>&zwnj;Desktop shortcuts&zwnj;</h3>
...
+                  <i class="material-icons">&#xE145;</i>
+                  <b class="btn btn-xs disabled">Print</b>
+                </td>
+                <td>&zwnj;Save a screenshot of a window to Pictures&zwnj;</td>
+              </tr>
+              <tr>
+                <td>
+                  <b class="btn btn-xs disabled">Print</b>
+                </td>
+                <td>&zwnj;Save a screenshot to Pictures&zwnj;</td>
```
Comment (fossfreedom) [inline data/gettingstarted.html]: "Pictures folder"

File: data/gettingstarted.html [hunk 7]
```diff
@@ -1233,6 +1237,46 @@ <h3>&zwnj;Desktop shortcuts&zwnj;</h3>
...
+                </td>
+                <td>&zwnj;Copy a screenshot of a window to clipboard&zwnj;</td>
+              </tr>
+              <tr>
+                <td>
+                  <b class="btn btn-xs disabled">&zwnj;Alt&zwnj;</b>
+                  <i class="material-icons">&#xE145;</i>
+                  <b class="btn btn-xs disabled">Print</b>
+                </td>
+                <td>&zwnj;Save a screenshot of a window to Pictures&zwnj;</td>
```
Comment (fossfreedom) [inline data/gettingstarted.html]: Pictures folder

File: data/gettingstarted.html (hunk 2 above)
Comment (manderso7) [reply data/gettingstarted.html]: Done, thank you.

File: data/gettingstarted.html (hunk 3 above)
Comment (manderso7) [reply data/gettingstarted.html]: Added, but can't figure out how to keep the list formatting without adding weird characters below  the line. Removed list formatting, and the button is in a silly location, maybe?

File: data/gettingstarted.html (hunk 4 above)
Comment (manderso7) [reply data/gettingstarted.html]: Added button, but can't get it to go to center. Thanks for idea.

File: data/gettingstarted.html (hunk 5 above)
Comment (manderso7) [reply data/gettingstarted.html]: Added $HOME/Pictures directory

File: data/gettingstarted.html (hunk 6 above)
Comment (manderso7) [reply data/gettingstarted.html]: Added $HOME/Pictures directory

File: data/gettingstarted.html (hunk 7 above)
Comment (manderso7) [reply data/gettingstarted.html]: Added $HOME/Pictures directory

File: data/gettingstarted.html (hunk 1 above)
Comment (manderso7) [reply data/gettingstarted.html]: Fixed.

**Comment by manderso7** [comment]
And I've uploaded the changed files to my forked repo, but don't know how to add/merge them here.

**Comment by fossfreedom** [comment]
Many thanks - I have pulled this into the eoan branch.  I'll merge this into the master branch later once the translators have got their hands on the new translation strings.

Cheers.  Much appreciated.

**Comment by manderso7** [comment]
Glad I could help, and thank you for your patience!

**Pull request closed without merging**